eth_keys/constants.py

```diff
@@ -14,6 +14,7 @@
 
 
 #
-# Jacobian Point at infinity
+# Jacobian Representation of the Point at Infinity
+# See: https://crypto.stackexchange.com/questions/41468/point-at-infinity-for-jacobian-coordinates/41473  # noqa: E501
```

Comment:
Woah, so I picked up about 10% of what was in that writeup. Definitely interested in someone else taking a look to confirm.